dynamic-plugins.default.yaml

```diff
@@ -546,6 +546,26 @@ plugins:
               clientId: "${KEYCLOAK_CLIENT_ID}"
               clientSecret: "${KEYCLOAK_CLIENT_SECRET}"
 
+  - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-cloud
+    disabled: true
+    pluginConfig:
+      # https://backstage.io/docs/integrations/bitbucketCloud/discovery#configuration
+      catalog:
+        providers:
+          bitbucketCloud:
+            yourProviderId: # identifies your ingested dataset
```

Comment:
do we really want to call the `yourProviderId`, or use `default`, as for `keycloak`: https://github.com/janus-idp/backstage-showcase/pull/681/files#diff-2a8ec8751c51963220c18905adc57407d70bbb6647e6fe394cc224291b93a11fR542

Reply:
I think `default` should fine fine.